react-components/src/components/Architecture/pointsOfInterest/useFilterPointsOfInterest.ts

```diff
@@ -23,7 +23,7 @@ function filterPoi(poi: PointOfInterest<unknown>, filter: PoiFilter | undefined)
   const lowerCaseContainsFilter = filter.contains.toLowerCase();
 
   return (
-    poi.properties.title?.toLowerCase().includes(lowerCaseContainsFilter) === true ||
+    poi.properties.title?.toLowerCase().includes(lowerCaseContainsFilter) ||
```

Comment:
just a cosmetic suggestion: maybe it is better to move these two statements to a var (before the return) and then return a var result instead.